README.md

```diff
@@ -51,7 +51,7 @@ Next, lets configure the development server. To keep things simple we can put th
 var hypernova = require('hypernova/server');
 
 hypernova({
-  devMode: true,
+  clustering: false,
```

Comment:
While I think this is a better name, "absence" and "false" should always be equivalent - and we want `clustering` to default to "true".

Can we think of an alternative name where it makes sense to default to `false`?